packages/applicant/src/pages/mandatory-questions/[mandatoryQuestionId]/organisation-summary.page.tsx

```diff
@@ -172,16 +172,18 @@ export default function MandatoryQuestionOrganisationSummaryPage({
             </dl>
 
             <Link
-              href={routes.api.mandatoryQuestions.createSubmission(
-                mandatoryQuestionId,
-                mandatoryQuestion.schemeId.toString()
-              )}
+              href={{
+                pathname: `/api/create-submission`,
```

Comment:
I do hate the routes file but maybe we should be consistent and use it here

Reply:
I genuinely think it's a bad idea and just makes the code harder to follow and read.